docs/usage.rst

```diff
@@ -50,6 +50,16 @@ plugins, so you can just use it:
        fs.create_file('/var/data/xx1.txt')
        assert os.path.exists('/var/data/xx1.txt')
 
+If you don't like the ``fs`` fixture name (``pylint`` may complain about the
```

Comment:
If you are bothered by PyLint warning,
`C0103: Argument name "fs" doesn't conform to snake_case naming style (invalid-name)`,
you can define a longer name in your ``conftest.py`` and use that in your tests: